hknweb/candidate/templates/candidate/candreq.html

```diff
@@ -57,7 +57,8 @@ <h3>Request Officer Challenge Confirmation</h3>
     </div>
 
     <!-- Right side of page, list of requested challenges -->
-    <div id='agenda'  style="width: 50%; float: right; padding-left: 20px; padding-right: 20px;">
+    <!--  TODO: refactor CSS (e.g. move into style.css and change px into em)  -->
+    <div id='agenda' style="width: 50%; float: right; padding-left: 20px; padding-right: 20px;">
```

Comment:
Is now a good time to do this todo? I imagine it'd look like
```css
#agenda {
    width: 50%;
    float: right;
    padding-left: 1.5em;
    padding-right: 1.5em;
}
```

But I'm not exactly sure what the em values are.

Reply:
Better than using px right?